### PR TITLE
Update to PHP 8.0+ syntax in `src/`, where viable

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All green 🎉

File: src/ProxyManager/Factory/RemoteObject/Adapter/BaseAdapter.php [hunk 1]
```diff
@@ -14,24 +14,16 @@
...
      *
      * @param array<string, string> $map map of service names to their aliases
      */
-    public function __construct(Client $client, array $map = [])
-    {
-        $this->client = $client;
-        $this->map    = $map;
+    public function __construct(
+        protected Client $client,
+        // Service name mapping
```
Comment (Ocramius) [inline src/ProxyManager/Factory/RemoteObject/Adapter/BaseAdapter.php:24]: This comment is not really needed

**Review by Ocramius — APPROVED** [review]
Thanks @samsonasik! Good refactoring (I suppose automated too? :D )

**Comment by samsonasik** [comment]
@Ocramius yes, that's by rector, do you want a github action for it? I may can create a PR for that.

**Comment by Ocramius** [comment]
@samsonasik IMO worth thinking about it for `laminas/` CI automation

**Pull request merged**